### PR TITLE
AENV-482: change alt tag for "go to live article" icon

### PR DESCRIPTION
**Pull request by thnkloud9**
tooltip update for preview / live view

### CODE REVIEW

**Comment by takeit** [comment]
:+1:

File: app/views/toolbar/mode.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <button
       type="button"
       class="btn btn-default btn-sm"
-      title="Preview article"
+      title="Go to live article"
```
Comment (takeit) [inline app/views/toolbar/mode.html]: this should be translatable :)

**Comment by thnkloud9** [comment]
added Translator.add("aes.btn.livearticle", "Go to live article", "messages", "en"); to translations, needs to be added to plugin translations as well.

**Comment by thnkloud9** [comment]
https://github.com/newscoop/Newscoop-Plugin-ArticleEditScreen/pull/31

**Comment by takeit** [comment]
:+1:

**Pull request merged**